package.json

```diff
@@ -1354,10 +1344,23 @@
                                             "type": "string"
                                         }
                                     },
+                                    "profiles": {
+                                        "type": "array",
+                                        "description": "%vscode-docker.tasks.docker-compose.dockerCompose.up.profiles%",
+                                        "items": {
+                                            "type": "string"
+                                        }
+                                    },
                                     "customOptions": {
                                         "type": "string",
                                         "description": "%vscode-docker.tasks.docker-compose.dockerCompose.up.customOptions%"
                                     }
+                                },
+                                "not": {
```

Comment:
Tasks in this screenshot: incorrect (both), correct (services), correct (profiles), correct (neither)
![image](https://user-images.githubusercontent.com/36966225/127540343-f5e6447c-ab89-4ed4-a0db-83e0e5cda8be.png)